### PR TITLE
ODS-5572 EdFi.Suite3.Ods.Api & EdFi.Suite3.Ods.Standard nuget package is missing with package reference for dependencies

### PR DESCRIPTION
**Pull request by semalaiappan**
image
after doing package reference in SK project and it test that it's working fine 
      <PackageReference Include="EdFi.Suite3.Ods.Api" Version="6.0.26" />
      <PackageReference Include="EdFi.Suite3.Ods.Common" Version="6.0.27" />
      <PackageReference Include="EdFi.Suite3.Ods.Standard" Version="6.0.14" />

### CODE REVIEW

**Review by bptillman — CHANGES_REQUESTED** [review]
Overall things look good here. Just a few changes to make and some and questions I have on some of the changes that were made.

File: .github/workflows/edFi.ods.api manual.yml [hunk 1]
```diff
@@ -29,20 +29,26 @@ jobs:
...
-        .\build.ps1 publish -InformationalVersion ${{ env.INFORMATIONAL_VERSION }} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}} -NuGetApiKey ${{ env.AZURE_ARTIFACT_NUGET_KEY }} -EdFiNuGetFeed ${{env.AZURE_ARTIFACT_URL}} -Solution "Application/EdFi.Ods.Api/EdFi.Ods.Api.sln" -ProjectFile "Application/EdFi.Ods.Api/EdFi.Ods.Api.csproj" -PackageName  "EdFi.Suite3.Ods.Api"
-      shell: pwsh      
+        .\build.ps1 publish -InformationalVersion ${{ env.INFORMATIONAL_VERSION }} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}} -NuGetApiKey ${{ env.AZURE_ARTIFACT_NUGET_KEY }} -EdFiNuGetFeed ${{env.AZURE_ARTIFACT_URL}} -ProjectFile "Application/EdFi.Ods.Api/EdFi.Ods.Api.csproj" -PackageName  "EdFi.Suite3.Ods.Api"
+      shell: pwsh
+    - name: Upload EdFi.Ods.Api Artifacts
+      if: success()
+      uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
+      with:
+        name: NugetPackages.Artifacts
+        path: ${{ github.workspace }}/NugetPackages/*.*
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply .github/workflows/edFi.ods.api manual.yml:54]: \Ed-Fi-ODS-Implementation\build.githubactions.ps1  & Ed-Fi-ODS\build.ps1

File: .github/workflows/edFi.ods.common manual.yml [hunk 2]
```diff
@@ -10,23 +10,60 @@ on:
...
 
     steps:
     - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
     - name: Setup .NET
       uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # 2.1.0
       with:
         dotnet-version: 6.0.x
+    - name: Support longpaths
+      run: git config --system core.longpaths true
+    - name: Checkout Ed-Fi-ODS-Implementation
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply .github/workflows/edFi.ods.common manual.yml]: I removed it but I still need BUILD_INCREMENTER: "17" to get new  EdFi.Suite3.Ods.Common Version="6.0.33" package

**Comment by semalaiappan** [comment]
image
EdFi.Ods.Extensions.Sk build succeed with latest package version

File: .github/workflows/edFi.ods.common manual.yml [hunk 3]
```diff
@@ -10,23 +10,60 @@ on:
 
 env:
   INFORMATIONAL_VERSION: "6.0"
-  BUILD_INCREMENTER: "1"
+  BUILD_INCREMENTER: "13"
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply .github/workflows/edFi.ods.common manual.yml]: still need BUILD_INCREMENTER: "17" to get new  EdFi.Suite3.Ods.Common Version="6.0.33" package

**Comment by semalaiappan** [comment]
<PackageReference Include="EdFi.Suite3.Ods.Api" Version="6.0.32" />
<PackageReference Include="EdFi.Suite3.Ods.Common" Version="6.0.33" />
<PackageReference Include="EdFi.Suite3.Ods.Standard" Version="6.0.22" />

**Review by bptillman — APPROVED** [review]

**Pull request merged**